cordova-plugin-outline/apple/src/OutlineVpn.swift

```diff
@@ -165,9 +166,12 @@ class OutlineVpn: NSObject {
         self.onStartVpnExtensionMessage(response, completion: completion)
       }
       var config: [String: String]? = nil
-      if connectionId != nil {
+      if !isAutoConnect {
         config = connection.config
         config?[MessageKey.connectionId] = connectionId
+      } else {
+        // macOS app was started by launcher.
+        config = ["is-on-demand": "true"];
```

Comment:
This constant is still here.  Could it be something like `MessageKey.isOnDemand`?

Reply:
Good catch, done.